src/bindings/js/node/tests/tensor.test.js

```diff
@@ -1,10 +1,27 @@
-var ov = require('../build/Release/ov_node_addon.node');
+// -*- coding: utf-8 -*-
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+let ov = require('../build/Release/ov_node_addon.node');
 const assert = require('assert');
 const { test, describe, it } = require('node:test');
+const random = require('random-bigint');
 
 const shape = [1, 3, 224, 224];
 const elemNum = 1 * 3 * 224 * 224;
 const data = Float32Array.from({length: elemNum}, () => Math.random() );
+const ets = [
```

Comment:
How can I decode `ets`?

Reply:
Those are parameters for tests, so I changed the name to `params`